source/features/mark-private-orgs.js

```diff
@@ -0,0 +1,44 @@
+import {h} from 'dom-chef';
+import select from 'select-dom';
+import {getCleanPathname} from '../libs/page-detect';
+import {getUsername} from '../libs/utils';
+import * as icons from '../libs/icons';
+import api from '../libs/api';
+
+export default async () => {
+	if (!getCleanPathname().startsWith(getUsername())) {
+		// Only for own user
+		return;
+	}
+	// List public orgs
+	const publicOrgs = [];
+	const orgDataList = await api(`users/${getUsername()}/orgs`, true);
+	if (!orgDataList) {
+		return;
+	}
+	for (const orgData of orgDataList) {
+		publicOrgs.push('/' + orgData.login);
+	}
+
+	// Display
```

Comment:
This comment is not useful. `Display` doesn't mean anything without any context.